Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN bundle install
 
 # Install Java, which is required for s3_website
 # Install texlive libraries, which are required for Pandoc
-RUN apt-get update && \
+RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list &&\
```

Comment:
Ah, yes, we've hit this issue with Jessie a few times! Sorry you ran into that.

I think the simpler fix is actually to update to a newer Debian version. In the `FROM` block at the type, I think you can switch to `ruby:2.6.2-stretch`.

Reply:
I removed the fix I included and tried again using ```ruby:2.6.2-stretch``` but it doesn't seem to fix the issue.

Reply:
What is the issue you see with `-stretch`? Did you re-run `docker-compose build`?

Reply:
It actually works. I had to remove the previous containers and images. I've updated the PR